### PR TITLE
AC: Added Open Images Dataset support

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

**Comment by Anna-Mironova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+"""
+
+from ..config import PathField
+from ..representation import DetectionAnnotation
+from ..utils import read_csv, check_file_existence
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class OpenImagesConverter(BaseFormatConverter):
+    __provider__ = 'open_images'
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py:24]: dataset support multiple tasks
```suggestion
    __provider__ = 'open_images_detection'
```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+        return ConverterReturn(annotations, {'label_map': label_map}, content_errors)
+
+    @staticmethod
+    def get_labels_mapping(labels_file):
+        labels = read_csv(labels_file, is_dict=False)
+        label_map = {}
+        reversed_annotation_label_map = {}
+        for idx, (label_name, real_name) in enumerate(labels):
+            label_map[idx + 1] = real_name
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py:85]: is it always starting from 1?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py:85]: you can add start=<number> parameter to enumerate for starting enumeration from specific point

**Review by eaidova — APPROVED** [review]

**Pull request merged**